Cluster related enhancements

dougm opened the pull request.

pietern commented on CHANGELOG.md:
```diff
@@ -2,6 +2,14 @@
 
 ### (unreleased)
 
+* Add object.HostSystem methods: EnterMaintenanceMode, EnterMaintenanceMode, Disconnect, Reconnect
```
2x EnterMaintenanceMode

pietern commented on govc/cluster/change.go:
```diff
@@ -0,0 +1,96 @@
+		string(types.DrsBehaviorFullyAutomated),
+	}
+	f.StringVar((*string)(&cmd.DrsConfig.DefaultVmBehavior), "drs-mode", "",
+		"DRS behavior for virtual machines"+strings.Join(drsModes, ", "))
```
Add ": " or something before strings.Join?

pietern commented on govc/cluster/change.go:
```diff
@@ -0,0 +1,96 @@
+		}
+
+		for _, cluster := range clusters {
+			fmt.Println(cluster.InventoryPath)
```
Debug or intentional?

pietern commented on govc/host/destroy.go:
```diff
@@ -0,0 +1,87 @@
+
+func (cmd *destroy) Register(f *flag.FlagSet) {}
+
+func (cmd *destroy) Process() error { return nil }
```
So this removes a host from a cluster? Could you add a little usage doc?

pietern commented:
Some minor comments, LGTM otherwise

dougm commented:
Thanks for the review @pietern - addressed all comments.  btw I'm tempted to rename host.destroy to host.remove, but stuck with 'destroy' as we use that name for govc calls to Destroy_Task.  Let me know if you'd prefer host.remove instead.

pietern commented:
@dougm Agree `host.remove` is better, since the host won't be destroyed at all ;)

Also, it's symmetric with `host.add`.

dougm commented:
@pietern host.remove it is

pietern commented:
LGTM! :+1:
